src/components/Markings/components/EditableRect.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react'
+import {
+    G,
+    Rect,
+    Circle,
+} from 'react-native-svg'
+import PropTypes from 'prop-types'
+import CloseButtonSVG from './CloseButtonSVG'
+
+export const RectCorners = {
+    upperLeft: 'upperLeft',
+    upperRight: 'upperRight',
+    bottomLeft: 'bottomLeft',
+    bottomRight: 'bottomRight'
+}
+
+class EditableRect extends Component {
+
+    /**
+     * Because Svgs don't plug in to react-native's animation engine,
+     * we have to update their props natively.
+     * 
+     * This function handles updating the props of the svgs of this component by
+     * taking an object of deltas
+     * 
+     * @param {An object that contains deltas to be applied to the shape} deltas
+     */
+    update({dx, dy, dw, dh}) {
+        this.rect.setNativeProps({
```

Comment:
I haven't seen this. but I will keep my eye out for it. I think we should leave it for now because I am doing some refactoring on this function in an upcoming PR